examples/tiff.html

```diff
@@ -0,0 +1,74 @@
+<html>
+    <head>
+        <title>Itowns - TIFF Parser</title>
+        <meta charset="UTF-8">
+        <link rel="stylesheet" type="text/css" href="css/example.css">
+        <link rel="stylesheet" type="text/css" href="css/loading_screen.css">
+
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <script src="js/GUI/dat.gui/dat.gui.min.js"></script>
+    </head>
+    <body>
+        <div id="viewerDiv"></div>
+        <script src="js/GUI/GuiTools.js"></script>
+        <script src="../dist/itowns.js"></script>
+        <script src="js/loading_screen.js"></script>
+        <script src="../dist/debug.js"></script>
+        <script type="text/javascript">
+            // # Simple Globe viewer
+
+            // Define initial camera position
+            var positionOnGlobe = { longitude: 1.411667, latitude: 43.6074422, altitude: 25000 };
+
+            // `viewerDiv` will contain iTowns' rendering area (`<canvas>`)
+            var viewerDiv = document.getElementById('viewerDiv');
+
+            // Instanciate iTowns GlobeView*
+            var view = new itowns.GlobeView(viewerDiv, positionOnGlobe);
+            setupLoadingScreen(viewerDiv, view);
+
+            function addTIFF() {
+                itowns.Fetcher.arrayBuffer('https://raw.githubusercontent.com/iTowns/iTowns2-sample-data/master/tiff/FLAG_T24.TIF')
+                    .then(itowns.TIFFParser.parse)
+                    .then(function _(texture) {
+                        texture.coords = new itowns.Coordinates('EPSG:4326', 1.4116667, 43.6074422);
```

Comment:
The `texture.coords` have bad name, because it's actually a `Extent`. 
this Extent allows you to place the texture on the tiles. In particular with initialization from parent texture. 

There are little changes to enhance the image display. 
* Use `Extent` insteaf of `Coordinates`, and add it a `zoom` property to set image scale.
* Add a layer property : use `imageParent` after zoom max to always display the ColorLayer after the Source.zoom.max.

Reply:
This needs another change in (`MaterialLayer` as well)[https://github.com/iTowns/itowns/blob/master/src/Renderer/MaterialLayer.js#L135].
> Add a layer property : use imageParent after zoom max to always display the ColorLayer after the Source.zoom.max.
I don't understand what you mean here

Reply:
A ColorLayer isn't displayed when Source.zoom.max is exceeded, it isn't the expected behaviour because the ColorLayer displaying shouldn't be controlled by the source.